af/fastpath/fastpath/core.py

```diff
@@ -124,13 +131,17 @@ def setup():
 
     # Run inside current directory in devel mode
     root = Path(os.getcwd()) if conf.devel else Path("/")
-    conf.conffile = root / "etc/fastpath.conf"
-    log.info("Using conf file %r", conf.conffile)
+    conf.conffile = root / "etc/ooni/fastpath.conf"
```

Comment:
Perhaps the `/` operator can be used in the rest of the path as well?

Reply:
"etc" / "ooni" / "fastpath.conf"? It's less readable/searchable and I don't see much benefit